src/org/folio/rest_v2/eureka/kong/Tenants.groovy

```diff
@@ -74,14 +75,16 @@ class Tenants extends Kong{
       .withClientSecret(retrieveTenantClientSecret(tenant))
   }
 
+  /**
+   * Retrieve Client Secret for the Tenant from AWS SSM parameter
+   * @param EurekaTenant object
+   * @return client secret as Secret object
+   */
   Secret retrieveTenantClientSecret(EurekaTenant tenant){
-    String userInput = context.input(
-      id: 'userInput'
-      , message: 'Please provide tenant Keycloak client password:'
-      , parameters: [context.string(name: 'InputValue', defaultValue: '', description: 'Keycloak client password')]
-    )
-
-    return Secret.fromString(userInput)
+    context.awscli.withAwsClient {
+      String clientSecret = context.awscli.getSsmParameterValue(Constants.AWS_REGION, tenant.secretStoragePathName)
+      return Secret.fromString(clientSecret)
```

Comment:
if (clientSecret == null || clientSecret.isEmpty()) {
        throw new IllegalStateException("SSM parameter value is null or empty for path: ${tenant.secretStoragePathName}")
    }

Reply:
It's not necessary because eventually, it will throw an error in any case.  No option to avoid it and doesn't need.